packages/react/src/global-nav/GlobalNav.css.ts

```diff
@@ -0,0 +1,19 @@
+import { recipe } from "../vanilla-extract";
```

Comment:
As there's no complex selectors or styling, we can inline the styling.

Reply:
Won't we need to pass className for the 'nav' ?

Reply:
It will be automatically handled when we are spreading the props for nav.